quick-install/configure-host.sh

```diff
@@ -2,7 +2,18 @@
 set -eo pipefail
 unset CDPATH
 
-registry_host="trow.kube-public"
+namespace='kube-public'
+if [ ! -z "$1" ]
```

Comment:
I suppose we should use proper argument handling now, just to avoid bugs later where the arguments get confused e.g. 
https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash#13359121

Reply:
I will try to add it

Reply:
done